### PR TITLE
Add disabling reaction pings

### PR DESCRIPTION
**Pull request by 0xlgs**
Some users may not like being pinged by bot accounts. Users trying to ping a system who disabled pings will get a message informing them of that, and a subsequent message from which they can copy paste the account's mention.

### CODE REVIEW

File: PluralKit.Core/Models/PKSystem.cs [hunk 1]
```diff
@@ -17,11 +17,12 @@ public class PKSystem
         [JsonIgnore] public string Token { get; set; }
         [JsonProperty("created")] public Instant Created { get; set; }
         [JsonProperty("tz")] public string UiTz { get; set; }
-        public PrivacyLevel DescriptionPrivacy { get; set; }
+        [JsonProperty("ping")] public bool Pings { get; set; }
+	public PrivacyLevel DescriptionPrivacy { get; set; }
```
Comment (silasary) [inline PluralKit.Core/Models/PKSystem.cs]: This line isn't indented correctly

Comment (0xlgs) [reply PluralKit.Core/Models/PKSystem.cs]: Whoops. Fixed.

**Comment by xSke** [comment]
Merging this now. I'm going to rename a few things and change it so it DMs them the error instead of posting it publicly, though.

**Comment by xSke** [comment]
Unsure why GitHub didn't detect the merge, but: https://github.com/xSke/PluralKit/commit/1d2995fe6f209a9a1612a9b8cc0b9a9d6a23550d

Gonna close the issue manually. Thanks! :)

**Pull request closed without merging**